Tests/Functional/Asynchronous/Expectations/main.swift

```diff
@@ -96,6 +96,21 @@ class ExpectationsTestCase: XCTestCase {
         XCTWaiter(delegate: self).wait(for: [foo, bar], timeout: 1)
     }
 
+// CHECK: Test Case 'ExpectationsTestCase.test_multipleExpectations_async' started at \d+-\d+-\d+ \d+:\d+:\d+\.\d+
+// CHECK: Test Case 'ExpectationsTestCase.test_multipleExpectations_async' passed \(\d+\.\d+ seconds\)
+    func test_multipleExpectations_async() async {
+        let foo = expectation(description: "foo")
```

Comment:
Nit: personal preference, I tend to prefer `first` and `second` as the expectation over `foo` and `bar` (less jargony and more importantly doesn't have the same connotation associated with "fubar").

Reply:
This just matches the style elsewhere in the file.